worker.mjs

```
@@ -301,6 +316,195 @@ function neighborhoodKeysWasm(lon, lat, radius) {
+  }
+  if (!Number.isFinite(rc.distance)) return { error: 'unreachable_in_corridor' };
+  const expanded = [rc.pathIdx[0]];
+  for (let i = 1; i < rc.pathIdx.length; i += 1) {
+    unpackChEdgeCsr(csr, rc.pathIdx[i - 1], rc.pathIdx[i], expanded);
+  }
+  const coordinates = [];
+  for (let i = 0; i < expanded.length; i += 1) {
+    const idx = expanded[i];
+    const lon = csr.lons[idx];
+    const lat = csr.lats[idx];
+    if (lon === lon && lat === lat) coordinates.push([lon, lat]); // NaN skip
```

Biome (2.4.15)

[error] This comparison uses the same expression on both sides.

(lint/suspicious/noSelfCompare)